.github/workflows/tests.yml

```diff
@@ -167,6 +167,7 @@ jobs:
           results-dir:  "${{ github.workspace }}/result/master"
       - name: Qodana - Code Inspection
         uses: JetBrains/qodana-action@cd7dc31b49427ce740ebf6255319b267801d08cf # renovate: tag=v3.2.1
+        continue-on-error: true
```

Comment:
Doesn't this make usage of qodana-action redundant? Even if it fails, we shall ignore the exit code and still move on with merging a PR.

Reply:
Not quite, as the print (short) step later does the actual checking and breaks the build.